app/scripts/filters/util.js

```diff
@@ -195,6 +195,11 @@ angular.module('openshiftConsole')
       return Constants.HELP[type] || Constants.HELP["default"];
     };
   })
+  .filter('apiLink', function(Constants) {
+    return function(type) {
```

Comment:
having a filter that calls into one of our API related services might be OK, but it definitely should not be pulling things from the Constants